spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -31,12 +24,6 @@
       it 'finishes successfully' do
         Sidekiq::Testing.inline! do
           expect { subject }.not_to raise_error
-          last_email = ActionMailer::Base.deliveries.last
-          expect(last_email.subject).to eq 'RRD claim - Offramped'
-          expect(last_email.body).to include submission.id
-          expect(last_email.body).to include 'API returned 24 medication requests'
-          expect(last_email.body).to include '2013: {"active"=>3}'
-          expect(last_email.body).to include '1998: {"active"=>4, "completed"=>2}'
         end
```

Comment:
Suggest checking for artifacts that the asthma processing was successful, like https://github.com/department-of-veterans-affairs/vets-api/blob/9ed57531c0960c8840f4c52df82023c7c0322add/spec/jobs/rapid_ready_for_decision/form526_hypertension_job_spec.rb#L76